### PR TITLE
fix: use int instead of double for timeout type.

### PR DESCRIPTION
**Pull request by bokuweb**
Hi :) I found FIXME comment in ops.rs. I think it is better to use `int` instead of `double` for timeout type in flatbuffer. Because delay is rounded by typescript side like following. If not, please reject this PR.

https://github.com/denoland/deno/blob/master/js/timers.ts#L180-L184

### CODE REVIEW

**Comment by piscisaureus** [comment]
For a *relative* time-out value int is probably sufficient (it means the max time-out value is about 24 days). Are you sure it's definitely not an absolute value (ms since epoch) that's being passed?

**Comment by bokuweb** [comment]
@piscisaureus Thanks for your review. Sure. I confirmed that it was not passed.
For example when following code executed,  `timeout` values are `3000`, `2998`.

```
setTimeout(() => console.log(1), 3000);
setTimeout(() => console.log(1), 6000);
```

**Comment by bokuweb** [comment]
@hayd OK :) Thanks for your proposal. I'll fix lt 👍

**Comment by bokuweb** [comment]
In Node.js if timeout value is larger than int32, emit warning and fire callback in next tick.

https://github.com/nodejs/node/blob/master/lib/internal/timers.js#L57-L66

Should we imitate it?

**Comment by hayd** [comment]
Yes. Good investigating!

**Comment by bokuweb** [comment]
@piscisaureus @hayd Thanks for your comments. I fixed.

File: js/timers.ts [hunk 1]
```diff
@@ -181,7 +185,15 @@ function setTimer(
   // and INT32_MAX. Any other value will cause the timer to fire immediately.
   // We emulate this behavior.
   const now = getTime();
+  if (delay > TIMEOUT_MAX) {
+    console.warn(
+      `${delay} does not fit into a 32-bit signed integer.
+Timeout duration was set to 1.`
```
Comment (ry) [inline js/timers.ts]: Please make it so there's not a line break in the template literal.

Comment (bokuweb) [reply js/timers.ts]: @ry Thanks 👍fixed.

**Comment by bokuweb** [comment]
@ry @piscisaureus Is there anything I can do in this PR?

**Comment by piscisaureus** [comment]
@bokuweb Looked good to me so I landed it. Thanks!

**Comment by bokuweb** [comment]
@piscisaureus Thanks for your review :)

**Pull request merged**